cmd/agent/subcommands/dogstatsdcapture/command.go

```diff
@@ -60,7 +60,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				fx.Supply(cliParams),
 				fx.Supply(core.BundleParams{
 					ConfigParams: config.NewAgentParamsWithoutSecrets(globalParams.ConfFilePath),
-					LogParams:    log.LogForOneShot("CORE", "off", true)}),
+					LogParams:    log.LogForOneShot(command.LoggerName, "off", true)}),
```

Comment:
This code is used at many locations
 
```
core.BundleParams{
					ConfigParams: config.NewAgentParamsWithoutSecrets(globalParams.ConfFilePath),
					LogParams:    log.LogForOneShot(command.LoggerName, "off", true)})
```
What do you think about creating a function that returns `core.BundleParams` for this case? If there are new modifications, we wouldn't have to apply the changes at many places.

Reply:
Yes exactly, I was on the fence about this wondering if it was worth it. I'll update the PR :+1: .